### DISCUSSION TITLE
GPIO not working

### DISCUSSION THREAD

**Discussion by FranticMan** [bigtreetech/CB1#60] [Q&A]
I have a BTT CB1 mounted on a P4B.
I have installed the latest Debian for CB1 mini.

I have tried a couple configurations but I wasn't able to get the GPIO running with my hardware.

Is there any software which works with the CM1 GPIO to setuo and control HI/LOW?
Is it a general problem or is there anybody for help?

I was trying with Python:
```
import os
os.environ['GPIOZERO_PIN_FACTORY'] = os.environ.get('GPIOZERO_PIN_FACTORY', 'mock')

from gpiozero import LED
from time import sleep

led1 = LED("17")
led2 = LED("27")
led3 = LED("22")
led4 = LED("13")

led1.blink()
led2.blink()
led3.blink()
led4.blink()

sleep(30000)
```

**Comment by Boztech3D** [comment]
Any luck on getting GPIO working?
